handlers/routes.go

```diff
@@ -8,10 +8,12 @@ func (s *Server) InitRoutes() {
 	s.Router.HandleFunc("/events/{id}", s.DeleteEvent).Methods("DELETE")
 
 	s.Router.HandleFunc("/categories", s.ListAllCategories).Methods("GET")
-	s.Router.HandleFunc("/categories/{id}", s.GetCategory).Methods("GET")
-	s.Router.HandleFunc("/categories", s.CreateCategory).Methods("POST")
-	s.Router.HandleFunc("/categories/{id}", s.UpdateCategory).Methods("PATCH")
-	s.Router.HandleFunc("/categories/{id}", s.DeleteCategory).Methods("DELETE")
+	/*
```

Comment:
Remove commented code

Reply:
Done :)

Reply:
@villaleo I'll need another approval after that last change, please